src/interpreter.ts

```diff
@@ -878,15 +878,21 @@ export class Interpreter<
       id,
       send: () => void 0,
       subscribe: (next, handleError, complete) => {
+        let unsubscribed = false;
         promise
           .then(response => {
+            if (unsubscribed) return;
             next && next(response);
+            if (unsubscribed) return;
```

Comment:
What's the use-case for this line?

Reply:
It's handling a case of consumer unsubscribing synchronously in the "next" handler - taking RxJS as example https://codesandbox.io/s/determined-rgb-bwdnn . "complete" callback should not be called in this case